source/fundamentals/logging.txt

```diff
@@ -0,0 +1,137 @@
+.. _csharp-logging:
+
+=======
+Logging
+=======
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+Starting in version 2.18, the {+driver-short+} uses the standard 
+`.NET logging API. <https://learn.microsoft.com/en-us/dotnet/core/extensions/logging?tabs=command-line>`__
+In this guide, you can learn how to use the driver to configure logging for your 
+application. 
+
+Configure Logging
+-----------------
+
+To specify the logging settings for your application, create a new instance of the 
+``LoggingSettings`` class, then assign it to the ``LoggingSettings`` property of your 
+``MongoClientSettings`` object.
+
+The ``LoggingSettings`` constructor accepts the following parameters:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 25 75
+
+   * - Property
+     - Description
+
+   * - ``LoggerFactory``
+     - | The ``ILoggerFactory`` object that creates an ``ILogger``. You can create 
+         an ``ILoggerFactory`` object by using the ``LoggerFactory.Create()`` method.
+       |
+       | **Data Type**: `ILoggerFactory <https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.iloggerfactory?view=dotnet-plat-ext-7.0>`__
+       | **Default**: null
+
+   * - ``MaxDocumentSize``
+     - | Optional. The maximum number of characters for extended JSON documents in logged 
+         messages.
+       |
+       | For example, when the driver logs the ``CommandStarted`` message, it truncates 
+         the ``Command`` field to the specified character limit.
+       |
+       | **Data Type**: {+int-data-type+}
+       | **Default**: 1000
```

Comment:
[question] In the Kafka docs, the default value is always stylized like `code`. There are no other examples I could find in the C# docs, so I don't know if this is a company standard or different based on the individual driver docs. Should this be styled as code?